code-challenges/leftJoin/left-join.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = (hm1, hm2) => {
+  let vals1 = hm1.data.map(bucket => bucket ? JSON.parse(bucket.toString()) : bucket).filter(bucket => bucket);
+  let vals2 = hm2.data.map(bucket => bucket ? bucket.toString() : bucket).filter(bucket => bucket);
```

Comment:
I'm fuzzy why this one has `bucket.toString()` and the other doesn't...?

This pattern is pretty gross; I'd try to find a way to at least teach your Hashtable to return all its values.